gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -126,4 +138,33 @@ public void flush()
       throws IOException {
     this.producer.flush();
   }
+  
+  private void provisionTopic(String topicName,Config config) {
+		String zooKeeperPropKey = KafkaWriterConfigurationKeys.KAFKA_TOPIC_CONFIG+KafkaWriterConfigurationKeys.CLUSTER_ZOOKEEPER;
```

Comment:
Instead of concatenating keys here, please change it in KafkaWriterConfigurationKeys to look like: 
public static final String CLUSTER_ZOOKEEPER = KAFKA_TOPIC_CONFIG + "zookeeper";